packages/styles/scss/ibm-dotcom-styles.scss

```diff
@@ -43,6 +43,7 @@
 @import 'patterns/blocks/content-group-simple/index';
 @import 'patterns/blocks/feature-card-block-medium/feature-card-block-medium';
 @import 'patterns/blocks/logo-grid/logo-grid';
+@import 'patterns/blocks/leadSpaceBlock/index';
```

Comment:
Folder name shouldn't be camel case. Rename to `leadspace-block`.

Reply:
renamed it.